pywinauto/windows/application.py

```diff
@@ -478,7 +478,14 @@ def cpu_usage(self, interval=None):
         if not self.process:
             raise AppNotConnected("Please use start or connect before trying "
                                   "anything else")
-        h_process = win32api.OpenProcess(win32con.MAXIMUM_ALLOWED, 0, self.process)
+
+        try:
+            h_process = win32api.OpenProcess(win32con.MAXIMUM_ALLOWED, 0, self.process)
+        except win32api.error as e:
+            if e.winerror == win32defines.ERROR_INVALID_PARAMETER:
+                raise ProcessNotFoundError('Process does not exist')
```

Comment:
It's better to add PID to error message.

Reply:
done